servicedirectory/api/ServiceDirectory.Samples/ResolveService.cs

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START servicedirectory_resolve_service]
+
+using Google.Cloud.ServiceDirectory.V1;
+
+public class ResolveServiceSample
+{
+    public Service ResolveService(string projectId = "my-project",
+        string locationId = "us-east1",
+        string namespaceId = "test-namespace",
+        string serviceId = "test-service")
+    {
+        // Create client
+        LookupServiceClient lookupServiceClient = LookupServiceClient.Create();
+        // Initialize request argument(s)
+        ResolveServiceRequest request = new ResolveServiceRequest
+        {
+            ServiceName = ServiceName.FromProjectLocationNamespaceService(projectId, locationId, namespaceId, serviceId),
+        };
+        // Make the request
+        ResolveServiceResponse response = lookupServiceClient.ResolveService(request);

```

Comment:
This should have an overload that simply takes the ServiceName as parameter? If there is one, please use it.

Reply:
I dont see an overload.  Using the request object